cypress/integration/accessibility_Table_spec.js

```diff
@@ -4,14 +4,17 @@ describe('Table Accessibility check', () => {
     cy.injectAxe();
   });
 
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip('Tests accessibility on the Table page', () => {
+  it('Tests accessibility on the Table page', () => {
     cy.configureAxe({
       rules: [
         {
           id: 'color-contrast',
           enabled: false,
         },
+        {
+          id: 'scrollable-region-focusable',
+          enabled: false,
+        },
```

Comment:
Added this to avoid confusion for the sticky header example

Reply:
@ayeshakmaz could you add a comment explaining why we disable it?